_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+`properties.marker-popup.popup-[SETTING]` in dash case. For example,
+this would configure the feature to use an "info" popup with a title
+and description text:
+
+    ```json
+    {
+      "type": "Feature",
+      "id": 984093,
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-110.948009,37.984995]
+      },
+      "properties": {
+        "marker-icon": {
+          "icon-base": "static-icon",
+          "icon-type": "info"
+        },
+        "marker-popup": {
+          "popup-base": "info-popup",
+          "popup-title": "Title here",
+          "popup-description": "Lorem ipsum dolor sit amet, consectetur adipisicing..."
+        }
+      }
+    }
+    ```
+
+# Styling `px-map-layer-geojson` GeoJSON data
+
+There are two ways to style the features that the px-map-layer-geojson draws.
+To style all the features for the entire layer, use the `featureStyle` attribute.
+To style each feature individually, add a `style` object to the feature's
+`properties`. The following style options are available:
+
+  - {Boolean} `stroke`: [default=true] Set to false to disable borders on polygons/circles
+  - {String} `color`: [default=$primary-blue] Color for polygon/circle borders
+  - {Number} `weight`: [default=2] Weight for polygon/circle borders in pixels
+  - {Number} `opacity`: [default=1.0] Opacity for polygon/circle borders
+  - {Boolean} `fill`: [default=true] Set to false to disable filling polygons/circles
+  - {String} `fillColor`: [default=$dv-light-blue] Color for polygon/circle fill
+  - {Number} `fillOpacity`: [default=0.4] Opacity for polygon/circle fill
+  - {String} `fillRule`: [default='evenodd'] Defines how the [inside of a shape](https://developer.mozilla.org/docs/Web/SVG/Attribute/fill-rule) is determined
+  - {String} `lineCap`: [default='round'] Defines the [shape to be used](https://developer.mozilla.org/docs/Web/SVG/Attribute/stroke-linecap) at the end of the stroke
+  - {String} `lineJoin`: [default='round'] Defines the [shape to be used](https://developer.mozilla.org/docs/Web/SVG/Attribute/stroke-linejoin) at the corner of a stroke
+  - {String} `dashArray`: [default=null] Defines the stroke [dash pattern](https://developer.mozilla.org/docs/Web/SVG/Attribute/stroke-dasharray)
+  - {String} `dashOffset`: [default=null] Defines the [distance into the dash to start the dash](https://developer.mozilla.org/docs/Web/SVG/Attribute/stroke-dashoffset)
+
+# Handling large volumes of data
+Px-map can handle large volumes of data, but there are limits. As you begin to
+approach these limits you may notice that the map takes longer to load data, or
+that panning becomes jerky. This is because there is overhead associated with
+loading data into the DOM and transforming its location when panning and zooming.
+
+## Requesting data with a bounding box query
+One method to minimize the effects of loading very large amounts of data is
+to, rather than requesting all the data for the entire map, just request the data
```

Comment:
This is an awkward aside. The verb tense should at least agree. I'm struggling with where the `is` and `to` belong... maybe this?
`One method to minimize the effects of loading very large amounts of data is, rather than to request all of the data for the entire map, to just request the data...`

Or maybe this instead, which seems clearer:
`Rather than requesting all of the data for the entire map, one method to minimize the effects of load times is to request just the data...`